src/contracts/Protocol/QuestionAPI.sol

```diff
@@ -109,11 +112,31 @@ contract QuestionAPI is Ownable, NFTLocked, FunctionLocked {
         return questionId;
     }
 
+    /**
+     * @notice Creates a challenge.
+     * @param uri The IPFS hash of the challenge.
+     * @return The challenge id
+     */
+    function proposeChallenge(string calldata uri) public returns (uint256) {
```

Comment:
confused by this - createChallenge seems to be gated but here we are doing something similar skipping the question creation going straight to minting without it being gated?

I'm also confused on the functionality where a question becomes a challenge 
What's the point of minting a question in createQuestion if both methods of creating and proposing a challenge to do the same thing - we are missing the step 
`function instantiateChallengeFromQuestion`

Reply:
so when you burn you just send 1000 metric to the treasury, the treasury being a burnable address